src/yang/ietf-diffserv.yang

```diff
@@ -71,21 +71,21 @@ module ietf-diffserv {
           "The dscp-max must be greater than or equal to dscp-min";
```

Comment:
Update description either to use the (new) name of the leaf, or remove the - to say 

"DSCP max must be greater than or equal to DCSP min value."